operations/services.md

```diff
@@ -1169,8 +1167,8 @@ Creates a new order with the specified products and items. Only positive charges
 | `ProductId` | string | required | Unique identifier of the [Product](services.md#product) to be ordered. |
 | `Count` | number | optional | Count of products to be ordered, e.g. 10 in case of 10 beers. |
 | `UnitAmount` | [Amount](services.md#amount-parameters) | optional | Unit amount of the product that overrides the amount defined in Mews. |
-| `StartUtc` | string | optional | Product start in UTC timezone in ISO 8601 format. For products with charging `Once` and `PerPerson` must be set to same value as `EndUtc`. |
-| `EndUtc` | string | optional | Product end in UTC timezone in ISO 8601 format. For products with charging `Once` and `PerPerson` must be set to same value as `StartUtc`. |
+| `StartUtc` | string | optional | Product start in UTC timezone in ISO 8601 format. For products with charging `Once` and `PerPerson` must be set to same value as `EndUtc`. Use only with operation [Add reservation](reservations.md#add-reservation) or [Add reservation product](reservations.md#add-reservation-product), can be omitted for [Add order](services.md#add-order) operation. |
```

Comment:
This is weird that we're forcing the partners to do redundant stuff like this (setting same start and end). It should rather be done the way, that you can specify end only for products charged per night. Otherwise, there should be check that end is empty. That way, it automatically filters all products of product service orders (they cannot be per night).

Reply:
And for product orders under product service orders, there should be another check that start is empty. We should have proper checks instead of forcing partners to send us dummy data. CC @patrikmiksovsky95 @vitasamek

Reply:
@siroky we aren't forcing anything. This is shared DTO between stay and nonstay order. For nonstay we have the consumption date which is equal to start in Commander (and therefore this start/end can be omitted)

Reply:
Oh, now I understand, You meant charged once for stay product. Yes, there this requirement is almost redundant. It makes code (and contract) simpler by requiring end once you send start.

Reply:
And on the other hand, if they shouldn't provide start or end, we should have checks that they don't provide. Not just let it be.

Reply:
I'm not against it, but that could be a breaking change for some partners. As long as they send same date for ConsumedUtc in order section and StartUtc/EndUtc in product section it's a valid submission (albeit redundant one, as you mentioned). Let's discuss it somewhere else 🙂

Reply:
It's really new property so faster we fix this, the less problems. Also I don't believe anyone started using these two properties for product service orders.